client/layout/style.scss

```diff
@@ -82,6 +82,18 @@
 			margin-top: 0 !important;
 		}
 	}
+
+	&.has-woocommerce-navigation {
+		#wpbody {
+			padding-left: 0;
+		}
+
+		#wpcontent {
+			> #wpbody {
+				margin-top: 32px !important;
```

Comment:
Instead of adding this, I think we could apply the above negative margin fix on `.woocommerce-admin-full-screen` to only pages with toolbars.

```
.wp-toolbar .woocommerce-admin-full-screen {
    margin-top: -$adminbar-height;
}
```

Reply:
I _think_ this is required because the block above is `!important` and needs to be overwritten. 

When looking at the OBW with Nav turned on, there is no `.wp-toolbar` so it continues to look off:

<img width="1025" alt="Screen Shot 2021-02-18 at 12 41 00 PM" src="https://user-images.githubusercontent.com/1922453/108282738-f9112800-71e6-11eb-997e-c7d82939f3c0.png">

Reply:
The margin should not be applied to the `#wpbody`.  The margin is added above like so:

```scss
	margin-top: -$adminbar-height;
	@include breakpoint( '<600px' ) {
		margin-top: -$adminbar-height-mobile;
	}
```

But my suggestion is to move that logic out and only apply it when the toolbar is present so that we don't need to apply margin on one element and then conditionally add negative margin on another:

```scss
.wp-toolbar .woocommerce-admin-full-screen {
	margin-top: -$adminbar-height;
	@include breakpoint( '<600px' ) {
		margin-top: -$adminbar-height-mobile;
	}
}
```

Tested and this works with the new nav on or off.